firmware/ec/src/devices/eth_sw.c

```diff
@@ -10,14 +10,15 @@
 //*****************************************************************************
 // Standard Header files
 //*****************************************************************************
-#include "inc/devices/eth_sw.h"
-
-#include "inc/devices/88E6071_registers.h"
+#include <driverlib/sysctl.h>
```

Comment:
Having this first causes issues when I try to compile.
Per the coding guidelines these should be in the following order:
```
   /* Module header[s] */
    …
    /* stdlib headers */
    …
    /* TI includes */
    …
    /* OC includes */
```

With each section alphabetical.

Reply:
Yeah, I had this problem when I merged.  It's fixed now.